src/constants/CurrencyFormatsEnum.ts

```diff
@@ -0,0 +1,7 @@
+export enum CurrencyFormats {
```

Comment:
```suggestion
export enum CurrencyFormat {
```

Reply:
We already have a `CurrencyFormat` component, which uses the "CurrencyFormats" enum, then we'd have to use aliases. I thought it would be easier like this as since it enumerate multiple formats... Any suggestion for another name or the alias we'd have to use if we use your suggestion here?